audio_service/example/lib/main.dart

```diff
@@ -776,264 +779,192 @@ class MediaLibrary {
   static const albumsRootId = 'albums';
 
   final items = <String, List<MediaItem>>{
-    AudioService.browsableRootId: [
+    AudioService.browsableRootId: const [
       MediaItem(
         id: albumsRootId,
-        album: "",
-        title: "Albums",
+        album: '',
+        title: 'Albums',
```

Comment:
Decided to search for an old issue about quotes:

https://github.com/dart-lang/site-www/issues/859

What a messy situation. BTW, I opt for double quotes here because they're English text.

Reply:
agree
btw above there are some log messages that mix both, so i didn't really pay attention to it